### ISSUE TITLE
npm: unable to verify the first certificate

### ISSUE THREAD

**Issue by Skinyi** [npm/npm#20949]
#### I'm opening this issue because:

  - npm is doing something I don't understand.

#### What's going wrong?

    Today i installed node.js by downloading and compiling the source code by myself and everything was  ok. But when i tried to install some npm modules like "forever", something unexpected occurs. The error messages are following:

> npm ERR! code UNABLE_TO_VERIFY_LEAF_SIGNATURE
> npm ERR! errno UNABLE_TO_VERIFY_LEAF_SIGNATURE
> npm ERR! request to https://registry.npmjs.org/forever failed, reason: unable to verify the first certificate

  I have tried to set the strict ssl false, but there will be another issue seems that i will accept a html file rather than a json file.

### supporting information:

 - `npm -v` prints: 5.6.0
 - `node -v` prints: 8.11.2
 - `npm config get registry` prints: https://registry.npmjs.org/
 - Windows, OS X/macOS, or Linux?: Linux, Cent OS
 - Network issues:
   - Geographic location where npm was run: 

      - I don't use a proxy, but have limited or unreliable internet access.

 - Container:

   -  I develop using Vagrant on OS X or Linux.

**Comment by legodude17** [comment]
Does this occur on the latest version of npm?

**Comment by Skinyi** [comment]
In fact，I`m not sure, i had downloaded node.js from its official website. It's the latest version.

**Comment by developer239** [comment]
@legodude17 How to resolve issue:

- ask npm version
- ask node version
- ask what OS is being used
- ask ... (fill in anything that will not really help but can make you look like you care about the guy trying to solve his issue 😄 )

**Comment by legodude17** [comment]
@developer239 What do you mean?

**Comment by Skinyi** [comment]
@developer239 Haha...,It`s completely true! Due to this terrible issue, i had to stop my work but after these days of waiting there was still noting valuable provided.

**Comment by legodude17** [comment]
@Skinyi This issue is often caused by incorrectly configured proxies or VPNs. Do you have either?

**Comment by Skinyi** [comment]
@legodude17 No, I don`t use any proxy or VPN. And I can directly visit [](https://registry.npmjs.org/forever) and receive a json type document via my firefox browser.

**Comment by developer239** [comment]
@legodude17 I believe I temporarily turned off SSL verification https://stackoverflow.com/a/40350239/4766136
